js_tutorial/exercises/002_arithmetic.js

```diff
@@ -0,0 +1,69 @@
+// Addition of two numbers
+
+let a , b;
+const addition = (a , b) => {
+    return (a + b);
+}
+console.log("The sum is " + addition(4 , 6));
+
+// Subtraction of two numbers
+
+const subtraction = (a , b) => {
+    return (a - b);
+}
+console.log("The difference is " + subtraction(19 , 9));
+
+// Multiplication of two numbers
+
+const product = (a , b) => {
+    return (a * b);
+}
+console.log("The product is " + product(45 ,2));
+
+// Division of two numbers (Quotient)
+
+const quotient = (a , b) => {
+    return (a / b);
```

Comment:
Comments should be right above the function definition. There's no gap needed between them.